skill-tree/src/main/java/com/RDS/skilltree/utils/GlobalExceptionHandler.java

```diff
@@ -61,4 +62,13 @@ public ResponseEntity<GenericResponse<Object>> handleException(Exception ex){
                 .status(HttpStatus.INTERNAL_SERVER_ERROR)
                 .body(new GenericResponse<>(null, "Something went wrong, please try again." ));
     }
+
+    @ExceptionHandler({InvalidParameterException.class})
```

Comment:
why this is not inside Exception folder..?

Reply:
I observed that under exception folder only exception classes were there and this file already was there in utils. I can move it to exception folder if you feel that's a better place for this! Since it was made by someone else I thought that is the right place. 

Also we could place it in utils/exceptions/ so things are more cleaner? Let me know and I'll move it